pkg/config/services.go

```diff
@@ -686,6 +686,18 @@ var SupportedServices = serviceConfigs{
 			regexp.MustCompile("firewall/(?P<FirewallName>[^/]+)"),
 		},
 	},
+	{
+		Namespace: "AWS/NetworkMonitor",
+		Alias: "networkmonitor",
```

Comment:
I can use some help to validate this: what is the official alias? I'm setting if to the service prefix string, not completely sure that is the case

Reply:
I think this is fine, alias is just a yace specific thing.